guidelines/index.html

```diff
@@ -531,6 +531,34 @@ <h2>Statement of Partial Conformance - Language</h2>
                 <p>A "statement of partial conformance due to language" may be made when the page does not conform, but would conform if <a>accessibility support</a> existed for (all of) the language(s) used on the page. The form of that statement would be, "This page does not conform, but would conform to WCAG 2.2 at level X if accessibility support existed for the following language(s):"</p>
             </section>
 
+            <section class="informative" id="privacy-summary">
+                <h2>Privacy Considerations</h2>
+                <p>Success Criteria within this specification that may relate to privacy are:</p>
```

Comment:
On the call we discussed having some mention that this was the list, as understood by the WG. Perhaps like:

Success Criteria within this specification which the Working Group has identified possible implications for privacy, either by providing protections for end users or which are important for web site providers to take in to consideration when implementing features designed to protect user privacy. This list reflects the current understanding of the Working Group but other Success Criteria may have privacy implications that the Working Group is not aware of at the time of publishing.